Makefile

```diff
@@ -100,7 +130,7 @@ list-platforms: ## - Show the possible PLATFORMS
 .PHONY: local
 local: ## - Build local binary for local environment (bin/fleet-server)
 	@printf "${CMD_COLOR_ON} Build binaries using local go installation\n${CMD_COLOR_OFF}"
-	$(if $(FIPS),GOEXPERIMENT=systemcrypto) go build $(if $(SNAPSHOT),-tags="snapshot",) $(if $(FIPS),-tags="requirefips",) -gcflags="${GCFLAGS}" -ldflags="${LDFLAGS}" -o ./bin/fleet-server .
+	$(if $(FIPS),GOEXPERIMENT=systemcrypto) go build -tags=${GOBUILDTAGS} -gcflags="${GCFLAGS}" -ldflags="${LDFLAGS}" -o ./bin/fleet-server .
```

Comment:
IIUC, `GOEXPERIMENT=systemcrypto` should be only set if `FIPS=true` or only when FIPS is set? I see `OEXPERIMENT=systemcrypto` even for FIPS=true, FIPS=false or FIPS=foo.

Shall we move `$(if $(FIPS),GOEXPERIMENT=systemcrypto)` to an env variable and be reused?